src/App.vue

```diff
@@ -33,6 +47,11 @@ export default defineComponent({
     );
 
     const store = useStore();
+
+    const isLoading = computed(() => store.getters.isLoading);
+    const showAlertMsg = computed(() => store.getters.showAlertMsg);
+    const alertMsg = computed(() => store.getters.alertMsg);
+
     const networkIdx = localStorage.getItem('networkIdx');
```

Comment:
saving the last network endpoint and not being able to change the network until it connects with a node may lead to infinite loading if the user chooses a local node and the UI fails to establish a connection. We might want to let the user switch networks even if the page is loading or failed to load.